SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -261,6 +261,14 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	FFilePath SpatialOSLaunchConfig;
 
 public:
+	/** Expose the runtime on a particular IP address when it is running on this machine. */
+	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (ConfigRestartRequired = false, DisplayName = "Expose local runtime"))
+	bool bExposeRuntimeIP;
+
+	/** If the runtime is set to be exposed, specify on which IP address it should be reachable. Changes are applied on next deployment startup. */
+	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (EditCondition = "bExposeRuntimeIP", ConfigRestartRequired = false, DisplayName = "Exposed local runtime IP address"))
+	FString ExposedRuntimeIP;
```

Comment:
The variables you add here should be initialized in USpatialGDKEditorSettings constructor

Reply:
Agreed: Added it now.